forest/cli/src/cli/snapshot_cmd.rs

```diff
@@ -323,6 +350,129 @@ fn clean(config: &Config, snapshot_dir: &Option<PathBuf>, force: bool) -> anyhow
     Ok(())
 }
 
+async fn validate(
+    config: &Config,
+    recent_stateroots: &i64,
+    snapshot: &PathBuf,
+    force: bool,
+) -> anyhow::Result<()> {
+    let confirm = if !force {
+        atty::is(atty::Stream::Stdin)
+            && Confirm::with_theme(&ColorfulTheme::default())
+                .with_prompt(format!(
+                    "This will result in using approximately {} MB of data. Proceed?",
+                    std::fs::metadata(snapshot)?.len() / (1024 * 1024)
+                ))
+                .default(false)
+                .interact()
+                .unwrap_or_default()
+    } else {
+        true
+    };
+
+    if confirm {
+        let tmp_db_path = TempDir::new()?;
+
+        let db_path = tmp_db_path.path().join(&config.chain.name);
+
+        #[cfg(feature = "rocksdb")]
```

Comment:
This does not look right here. The cli shouldn't have features for rocksdb and paritydb.